docker-files/docker/app/Dockerfile

```diff
@@ -41,8 +41,8 @@ RUN echo "deb http://ppa.launchpad.net/ondrej/php/ubuntu xenial main" > /etc/apt
 RUN ln -sf /dev/stdout /var/log/nginx/access.log \
     && ln -sf /dev/stderr /var/log/nginx/error.log
 
-COPY h5bp /etc/nginx/h5bp
 COPY default /etc/nginx/sites-available/default
+COPY h5bp /etc/nginx/h5bp
```

Comment:
Is there any specific need to change the order of this line?

Reply:
This line change is not needed. Was used as a cache busting trick to rebuilt from that step onward instead of rebuilding the whole image.